### PR TITLE
[Issue #108] Matching public image families and adding Ubuntu 16.04

### PR DESCRIPTION
**Pull request by nelsonjr**
Addresses https://github.com/chef/knife-google/issues/108

### CODE REVIEW

File: lib/chef/knife/cloud/google_service.rb [hunk 1]
```diff
@@ -66,9 +66,13 @@ class GoogleService < Service
       "sles-11"            => { project: "suse-cloud",        prefix: "sles-11" },
       "sles-12"            => { project: "suse-cloud",        prefix: "sles-12" },
       "ubuntu-12-04"       => { project: "ubuntu-os-cloud",   prefix: "ubuntu-1204-precise" },
+      "ubuntu-1204-tls"    => { project: "ubuntu-os-cloud",   prefix: "ubuntu-1204-precise" },
```
Comment (erjohnso) [inline lib/chef/knife/cloud/google_service.rb]: s/tls/lts (long term support)

Comment (nelsonjr) [inline lib/chef/knife/cloud/google_service.rb]: Nice catch, thanks! Too much security related work for me I guess :-)

**Comment by nelsonjr** [comment]
Fixed. PTAL.

**Comment by erjohnso** [comment]
lgtm - thanks for the update

**Pull request merged**